docs/docs/api/Tasks/fetchTaskByDay.md

```diff
@@ -7,69 +7,56 @@ borderRadius: '2px',
 color: '#E83E8C',
 }}>{children}</span> );
 
-<Endpoint>GET tasks/day/:day/</Endpoint>: Fetching a single Task
+<Endpoint>GET /tasks/day/:day</Endpoint>: Fetching a single Task
 
 ```json
-{
-
-}
+{}
 ```
 
 ### Example Request
+
 This is a **protected route**, a **valid JWT is required** in the header field
+
 #### Header
+
 ```
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
 #### Body
-```json
-{
 
-}
+```json
+{}
 ```
 
 #### Parameters
+
 ```json
 {
-    day: "yyyy-MM-dd'T'HH:mm:ss" // ISO formatted START date
+  "day": "yyyy-MM-dd'T'HH:mm:ss" // ISO formatted START date
 }
-
 ```
 
 ### Possible Responses
+
 #### Immediate Success
+
 ```json
 {
-	"code": 500,
-	"message": "Could not fetch doc",
-	"data": {}
+  "code": 200,
```

Comment:
Similar to my comment in the fetch single task markdown file, please change these to the correct status that we're returning from the API. The template that I provided was just to show an example of how my API happened to return data, but in our case, since we're returning different information it will have to be changed.

Reply:
Again, please make the changes to any of the other files that happen to not display the correct failure/success response